sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveComparisonTest.scala

```diff
@@ -296,7 +296,7 @@ abstract class HiveComparisonTest
 
             val hiveQueries = queryList.map(new TestHive.HiveQLQueryExecution(_))
             // Make sure we can at least parse everything before attempting hive execution.
-            hiveQueries.foreach(_.logical)
+            hiveQueries.foreach(_.analyzed)
```

Comment:
This change breaks golden answer generation.  You can't analyze queries unless you are also executing DDL from previous queries.